tests/integration/courseSlots.test.js

```diff
@@ -494,7 +494,7 @@ describe('COURSE SLOTS ROUTES - PUT /courseslots/{_id}', () => {
       expect(response.statusCode).toBe(403);
     });
 
-    it('should return 403 if try to update intra_holding course slot from other holding', async () => {
+    it('should return 403 if try to update course slot from other holding (intra_holding)', async () => {
```

Comment:
L'intitulé me semble pas clair ici non plus.
Que penses-tu de : "should return 403 if user from other holding try to update course slot"

Reply:
done